src/app/work-item/work-item-board/work-item-board.component.html

```diff
@@ -38,8 +38,9 @@
                     </div>
                   </div>
                   <div [dragula]='"first-bag"' class="card-wrapper">
-                    <div class="board-card flex-container in-column-direction flex-grow-1" *ngFor='let item of lane.workItems' [attr.data-id]="item.id"
-                         (mousedown)="getWI(item)" (touchstart)="getWI(item)">
+                    <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select"
```

Comment:
@sanbornsen you need to add the class `board-card` back to have the card content align properly

with `board-card`:
![screen shot 2017-04-19 at 3 43 21 pm](https://cloud.githubusercontent.com/assets/4032718/25198912/f24ac8d0-2516-11e7-8c3f-f791d20075bd.png)